okhttp-urlconnection/src/main/java/okhttp3/internal/huc/OkHttpURLConnection.java

```diff
@@ -54,7 +54,6 @@
 import okhttp3.internal.Version;
 import okhttp3.internal.http.HttpDate;
 import okhttp3.internal.http.HttpHeaders;
-import okhttp3.internal.http.HttpMethod;
```

Comment:
(particularly the part where it looks like we now always send a content-type, even on GET requests)